### ISSUE TITLE
Default Temperature values repeated in preferences menu with screen readers

### ISSUE THREAD

**Issue by Nancy-Salpepi** [phetsims/greenhouse-effect#359]
For https://github.com/phetsims/qa/issues/973, when I change the default temperature units using the arrow keys, the screen reader (a11y view, VO, NVDA) will state the default temperature twice.

I know that for this release, the sim itself doesn't have interactive description, but according to https://github.com/phetsims/quadrilateral/issues/441 I should be able to navigate the preferences menu with a screen reader when a sim has alt input.

<img width="641" alt="Screenshot 2023-08-29 at 1 49 38 PM" src="https://github.com/phetsims/greenhouse-effect/assets/87318828/da135469-b797-485b-bda9-8cc465a42d31">

**Comment by jessegreenberg** [comment]
Hey @arouinfar, yes there was a decision to make Preferences contents screen reader accessible for all sims that support alternative input. I apologize that wasn't communicated well. Feel free to reassess that decision for this sim, I would hate to hold up the release if this is the only issue. It may be quick to fix though.

**Comment by arouinfar** [comment]
Thanks @Nancy-Salpepi. 

@jbphet in my opinion, this is something we can defer until we publish Greenhouse Effect with full description. The Default Temperature Units control is usable with a screen reader, even with the repeated responsive description.

> Yes there was a decision to make Preferences contents screen reader accessible for all sims that support alternative input. 

Thanks for confirming @jessegreenberg. I understand the motivation, but this seems somewhat arbitrary to me. I question the utility of state description in the Preferences dialog, but no state description elsewhere in the sim.

> I apologize that wasn't communicated well. Feel free to reassess that decision for this sim.

Luckily @Nancy-Salpepi has a good memory. :smile:

That said, this actually affects quite a few other sims in the pipeline such as Acid-Base Solutions, Center and Variability, and Kepler's Laws. None of these sims currently have any description content for the sim-specific Preferences. 

If this is the new norm, there should be a PSA in dev meeting (and maybe on the dev-public Slack channel) and process documentation should be updated accordingly.

>  I would hate to hold up the release if this is the only issue. It may be quick to fix though.

Greenhouse Effect will need to go through a spot-check, so I'll defer to @jbphet to decide whether or not he wants to address it for this release or defer until we publish with full description.

**Comment by jessegreenberg** [comment]
Sounds good! I just sent out a PSA about this to a the dev-public and design-sims slack channels.

**Comment by jbphet** [comment]
I think this is a bug, but it has been a little misinterpreted.  This is _not_ a double announcement of the change to the default temperature units.  Note that the message isn't about the temperature units, it is about actual temperature _values_.  What is really going on here is that the change to the default temperature units is triggering a change to the chosen units in the first two screens, and their "alerters" announce the temperature using the new units.  The only reason you don't see such an announcement for all three screens is that the third screen hasn't had much description added yet.

Another way this bug manifests is that if you change the default units when viewing one of the first two screens you'll also see two alerts.

I've addressed this by adding a feature to the gas concentration alerter that prevents it from making alerts if the Scenery node associated with it is not visible.  This seems to work pretty well.

**Comment by jbphet** [comment]
@jessegreenberg - The fix I've described in the previous comments and that can be seen in the commits for this issue may be something that should be generalized to the `Alerter` class.  Since you're the primary author on `Alerter`, I think this should be your call.  Please take a look and see if you think a way to enable/disable in the base class is appropriate.  If so, we should probably log a separate issue and I'd be happy to work with you to move this code into the base class.

**Comment by jbphet** [comment]
I created a separate issue for the question of whether this should be generalized, see https://github.com/phetsims/scenery-phet/issues/819.  I'll unassign @jessegreenberg and will have QA check this issue on the next RC.

**Comment by jbphet** [comment]
QA - feel free to close once you've verified that there are no more messages from the home screen and only a single message when the units are changed when viewing each of the first two screens.

**Comment by Nancy-Salpepi** [comment]
In rc.2 with VO, I do not hear any temperature values when on the home screen and I only hear values once when switching between default units. 
I will leave open for @KatieWoe to take a look with NVDA/JAWS.

**Comment by KatieWoe** [comment]
NVDA looks ok to me.
